ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/run_synthea_locally.sh

```diff
@@ -14,6 +14,9 @@ help() {
   echo "--synthea_jar, -j       : boolean (true/false) indicating to use Synthea Release jar file (default true)"
   echo "--cleanup, -c           : boolean (true/false) whether to perform 'pre' and 'post' file cleanup (default true)"
   echo "--num_future_months, -f : number of months into the future claims can have their claim dates set to (default 0)"
+  echo "--target_contract, -tc  : indicates a partD contract to tie all generated benes to if --use_target_contract is set to true"
+  echo "--use_target_contract, -utc : If set to true, indicates to tie all generated items to a single partD contract specified by --target_contract, must be 5 characters (default Y9999)"
```

Comment:
A multi-character flag with a single dash is confusing to me.  I'm accustomed to seeing a single dash followed by multiple characters to be a combination of several single character flags.  Maybe use `--utc` and `--tc` instead to be consistent with common practice.